template-manager-cmtemplate/src/main/resources/definitions/cm-cloud-storage-location-specification.json

```diff
@@ -87,6 +87,17 @@
       "relatedServices": ["PROFILER_ADMIN_AGENT", "PROFILER_METRICS_AGENT", "PROFILER_SCHEDULER_AGENT"],
       "requiredForAttachedCluster": true,
       "supportedStorages": ["ADLS", "WASB", "S3", "GCS", "ADLS_GEN_2"]
+    },
+    {
+      "type": "DEFAULT_FS",
+      "propertyName": "core.defaultFs",
```

Comment:
Based on https://jira.cloudera.com/secure/attachment/231501/231501_out.diff (referenced from the Jira of this PR) the name should be different! I.e. core_defaultfs